### PR TITLE
Even Quickerstart [VS-682]

### PR DESCRIPTION
**Pull request by mcovarr**
[Link to successful run](https://app.terra.bio/#workspaces/gvs-dev/mlc%20GVS%20Quickstart%20v3/job_history/7f0ec754-9739-40c4-b877-34b45808852b)

### CODE REVIEW

File: scripts/variantstore/wdl/GvsPrepareRangesCallset.wdl [hunk 1]
```diff
@@ -114,7 +114,7 @@ task PrepareRangesCallsetTask {
   }
 
   runtime {
-    docker: "us.gcr.io/broad-dsde-methods/variantstore:2023-06-01-alpine"
+    docker: "us.gcr.io/broad-dsde-methods/variantstore:2023-06-05-alpine-bc1f7a5a2"
```
Comment (RoriCremer) [inline scripts/variantstore/wdl/GvsPrepareRangesCallset.wdl:117]: now that we are going to updating these multiple times a pr---would it make sense to have a constant somewhere?

File: scripts/variantstore/wdl/GvsQuickstartIntegration.wdl [hunk 2]
```diff
@@ -4,14 +4,43 @@ import "GvsQuickstartVcfIntegration.wdl" as QuickstartVcfIntegration
 import "GvsQuickstartHailIntegration.wdl" as QuickstartHailIntegration
 import "GvsUtils.wdl" as Utils
 
+
+task FilterIntervalListChromosomes {
```
Comment (RoriCremer) [inline scripts/variantstore/wdl/GvsQuickstartIntegration.wdl:8]: nit: you say it below, but a comment here about what you are doing would be nice

**Review by RoriCremer — APPROVED** [review]

File: scripts/variantstore/wdl/GvsQuickstartVcfIntegration.wdl [hunk 3]
```diff
@@ -7,52 +7,32 @@ workflow GvsQuickstartVcfIntegration {
 
     input {
         String branch_name
+        File interval_list
+        String expected_output_prefix
         Boolean use_classic_VQSR = true
         Boolean extract_do_not_filter_override = true
-        String expected_output_prefix = "gs://gvs-internal-quickstart/integration/2023-05-31/"
 
         Array[String] external_sample_names = [
```
[the remark this answers is not in the document]
Comment (mcovarr) [reply scripts/variantstore/wdl/GvsQuickstartVcfIntegration.wdl:15]: no

File: scripts/variantstore/wdl/GvsPrepareRangesCallset.wdl (hunk 1 above)
Comment (mcovarr) [reply scripts/variantstore/wdl/GvsPrepareRangesCallset.wdl:117]: I don't understand what you mean, happy to discuss internally

**Review by koncheto-broad — APPROVED** [review]
LGTM.  Maybe more general than it needs to be (vs just feeding it an interval list that was manually cut down to the regions we want, which is definitely what I did in my small dev workspace), but that gives us flexibility in the future so it isn't a bad thing

**Comment by mcovarr** [comment]
> Do we really want the additional complexity of the filtered interval list if it doesn't significantly reduce the amount of time the workflow takes to run?

Fair question, we can discuss in standup.

**Pull request merged**